docs/guides/features/system-tray.md

```diff
@@ -163,9 +163,9 @@ fn main() {
     .system_tray(SystemTray::new().with_menu(tray_menu))
     .on_system_tray_event(|app, event| match event {
       SystemTrayEvent::MenuItemClick { id, .. } => {
-        // get a handle to the clicked menu item
+        // get a handle on the clicked menu item
```

Comment:
```suggestion
        // get a handle to the clicked menu item
```

The handle points to the clicked menu item

Reply:
I'm unable to confirm which variant is correct. 
I looked into the definition of prepositions, but their use in the current context (programming) might differ slightly from what the norm is. 
I believe that either of them works in the currently abstract context.